test/DynamoCoreTests/NodeToCodeTest.cs

```diff
@@ -1191,6 +1191,36 @@ public void TestNodeToCodeUndoRecorder()
             Assert.AreEqual(0, recorder.ActionCount());
         }
 
+        [Test]
+        public void TestNodeToCodeStringInputEscaping()
+        {
+            // Arrange
+            OpenModel(@"core\node2code\stringNodesInNeedOfEscaping.dyn");
+            var nodes = CurrentDynamoModel.CurrentWorkspace.Nodes;
+            var engine = CurrentDynamoModel.EngineController;
+
+            // Act
+            var nodesToCode = engine.ConvertNodesToCode(nodes, nodes);
+            var results = nodesToCode.AstNodes.OfType<BinaryExpressionNode>()
+                .Where((x, i) => i < 8)
+                .Select(x => x.RightNode.ToString())
+                .ToList();
+
+            // Assert
+            var expect = new List<string>
```

Comment:
Nice, makes sense now!

If we are really picky the `except` list should probably go into the `Arrange` section